plugins/Diagnostics/Diagnostic/ConfigInformational.php

```diff
@@ -0,0 +1,81 @@
+<?php
+/**
+ * Matomo - free/libre analytics platform
+ *
+ * @link https://matomo.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ */
+namespace Piwik\Plugins\Diagnostics\Diagnostic;
+
+use Piwik\ArchiveProcessor\Rules;
+use Piwik\CronArchive;
+use Piwik\Development;
+use Piwik\Option;
+use Piwik\Plugin\Manager;
+use Piwik\SettingsPiwik;
+use Piwik\Translation\Translator;
+
+/**
+ * Informatation about the Matomo configuration
+ */
+class ConfigInformational implements Diagnostic
+{
+    /**
+     * @var Translator
+     */
+    private $translator;
+
+    public function __construct(Translator $translator)
+    {
+        $this->translator = $translator;
+    }
+
+    public function execute()
+    {
+        $results = [];
+
+        if (SettingsPiwik::isMatomoInstalled()) {
+            $results[] = DiagnosticResult::informationalResult('Browser Archiving Enabled', Rules::isBrowserTriggerEnabled());
```

Comment:
fyi we don't really want to translate them likely cause it's mostly information for us... makes it easier to analyze things having it in english